index.js

```diff
@@ -72,7 +72,12 @@ Options:
         console.error(`  ${red('Remember: ')} If you have some money to spare, donate it to twist.moe so they can the servers up and running!`)
         try{
             const donation = await getJSON('/api/donation')
-            console.error(`\n  They're ${red(Math.round((donation.remaining + Number.EPSILON)*100)/100 + '$ short')} on money this month.\n  To donate, please visit https://twist.moe/\n`)
+            const donationAmount = Math.round((donation.remaining + Number.EPSILON)*100)/100;
+            if(donationAmount >= 0){
+                console.error(`\n  They're ${red('$'+donationAmount + ' short')} on money this month.\n  To donate, please visit https://twist.moe/\n`)
+            }else{
+                console.error(`\n  They're ${green('$'+-donationAmount + ' over')} the donation goal this month.\n  To donate, please visit https://twist.moe/\n`)
+            }
```

Comment:
These are just minor code-style changes

```suggestion
            const donationAmount = Math.round((donation.remaining + Number.EPSILON)*100)/100;
            if(donationAmount >= 0)
                console.error(`\n  They're ${red('$'+donationAmount + ' short')} on money this month.\n  To donate, please visit https://twist.moe/\n`)
            else
                console.error(`\n  They're ${green('$'+-donationAmount + ' over')} the donation goal this month.\n  To donate, please visit https://twist.moe/\n`)
```